### PR TITLE
MEPTS-901 | list of patients defaulters or iit

### PR DESCRIPTION
**Pull request by helderjosue**

### CODE REVIEW

**Review by ItsMurumba — CHANGES_REQUESTED** [review]
1. Age: This is based on endDate and what we are currently using is based on the current date. Check List of Children Dataset.
2. Add Parameters to the **constructDataSet** DatasetDefinition.
cc @helderjosue @rupansana12

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ListOfPatientsDefaultersOrIITCohortQueries.java [hunk 1]
```diff
@@ -1613,7 +1613,7 @@ public DataDefinition getPatientsMarkedYes() {
   }
 
   /** @return sqlCohortDefinition */
-  public DataDefinition getStateProvince() {
+  public DataDefinition getLocation() {
 
     SqlPatientDataDefinition sqlPatientDataDefinition = new SqlPatientDataDefinition();
     sqlPatientDataDefinition.setName("All patients marked as: Sim");
```
Comment (ItsMurumba) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ListOfPatientsDefaultersOrIITCohortQueries.java]: Update the setName to Location

**Review by ItsMurumba — CHANGES_REQUESTED** [review]
In the cohort queries, supply the correct datadefibition set name for the following:

1. getLocation()
2.  getContact()
3. getNeighborhood()
4. getReferencePoint()
5. getPatientsMarkedNo()

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ListOfPatientsDefaultersOrIITCohortQueries.java (hunk 1 above)
Comment (helderjosue) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ListOfPatientsDefaultersOrIITCohortQueries.java]: All implemented

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/data/definition/CurrentDateDataDefinition.java [hunk 2]
```diff
@@ -0,0 +1,3 @@
+package org.openmrs.module.eptsreports.reporting.data.definition;
+
+public class CurrentDateDataDefinition {}
```
Comment (cfaife) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/data/definition/CurrentDateDataDefinition.java]: is this class in use?

**Review by cfaife — CHANGES_REQUESTED** [review]

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/data/definition/CurrentDateDataDefinition.java (hunk 2 above)
Comment (rupansana12) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/data/definition/CurrentDateDataDefinition.java]: we are not using, I'm going to remove.

**Review by cfaife — APPROVED** [review]

**Review by ItsMurumba — APPROVED** [review]

**Pull request merged**